mscore/musescore.cpp

```diff
@@ -6018,6 +6018,32 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   cv->setMag(MagIdx::MAG_100, 1.0);
             setMag(1.0);
             }
+      else if (cmd == "zoom-page-width") {
+            if (cv) {
+                  // Store current zoom-level statically here for later retrieval
+                  // instead of implementing it as a member variable since there's
+                  // (currently) no other use for it.
+                  MagIdx current_idx = MagIdx(mag->currentIndex());
```

Comment:
We use camel case for variables naming across the code so this variable should rather be named `currentIdx` instead, see the [handbook](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#Names).